mmedit/models/editors/stylegan2/stylegan2_generator.py

```diff
@@ -161,10 +196,10 @@ def __init__(self,
         self.upsamples = nn.ModuleList()
         self.to_rgbs = nn.ModuleList()
 
-        in_channels_ = self.channels[4]
+        blk_in_channels_ = self.channels[4]
```

Comment:
You may give ``blk_in_channels_`` a description so that user can know its meaning.